File: Taller Tipos de clases/Go/Clase Abstracta/Dispositivo.go
abstract class Dispositivo
{
    public string Marca;

    public Dispositivo()
    {
        Marca = "NA";
    }

    public Dispositivo(string marca)
    {
        Marca = marca;
    }

    public Dispositivo(Dispositivo d)
    {
        Marca = d.Marca;
    }
}

File: Taller Tipos de clases/JAVA/Clase Abstracta/Dispositivo.java
abstract class Dispositivo {
    String marca;

    // Constructor sin parámetros
    public Dispositivo() {
        marca = "NA";
    }
 // Constructor con parámetros
    public Dispositivo(String marca) {
        this.marca = marca;
    }
    // Constructor copia
    public Dispositivo(Dispositivo d) {
        this.marca = d.marca;
    }
}
